packages/flutter_webrtc/pubspec.yaml

```diff
@@ -0,0 +1,29 @@
+name: flutter_webrtc_tizen
+description: Flutter WebRTC plugin for Tizen, based on GoogleWebRTC.
+repository: https://github.com/flutter-tizen/plugins/tree/master/packages/flutter-webrtc
+version: 0.9.11
+
+flutter:
+  plugin:
+    platforms:
+      tizen:
+        pluginClass: FlutterWebRtcTizenPlugin
+        fileName: flutter_webrtc_tizen_plugin.h
+
+dependencies:
+  dart_webrtc: ^1.0.11
+  flutter:
+    sdk: flutter
+  path_provider: ^2.0.2
+  webrtc_interface: ^1.0.10
+
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+  import_sorter: ^4.6.0
+  lints: ^2.0.0
+  pedantic: ^1.11.1
```

Comment:
And you can remove all these unused dependencies (except `flutter`). Consider adding `path_provider_tizen` as a dependency of this package instead.

Reply:
Do we need add path_provider_tizen in pubspec.yaml ?

Reply:
Maybe. You can leave it as it is.